src/cmdstan/stansummary_helper.hpp

```diff
@@ -50,8 +52,7 @@ int compute_precision(double value, int sig_figs, bool scientific) {
 }
 
 int calculate_column_width(const Eigen::VectorXd &x, const std::string &name,
```

Comment:
Could we give these different names? I just noticed there are `calculate_column_width` and `calculate_column_widths`

Reply:
will do.

Reply:
done.